app/ui-account/client/accountProfile.js

```diff
@@ -60,6 +60,7 @@ const loginWith = function(event, template) {
 			}
 			return toastr.error(error.message);
 		}
+		callbacks.run('onUserLogin');
```

Comment:
`app/ui-account/client/accountProfile.js` and `app/ui-account/client/avatar/prompt.js` files are being removed in the upstream as they are refactoring to react. 

So need to make respective changes in new react code

Reply:
This callback is managed separately which I guess can be used with react as well, so once it's done we can migrate it based on the new implementation

Reply:
This call back is done to make sure that the user profile is linked to the subscription doc in all cases

Reply:
We have already merged react implementation while upstream catchup. That's why I'm saying, so we don't miss these changes. 😄 
You can see conflicts below, those are for these only. Can you please resolve it?